src/js/views/patients/worklist/role_component.js

```diff
@@ -0,0 +1,49 @@
+import Radio from 'backbone.radio';
```

Comment:
This could potentially live in the filter_views file, but I thought for the same of readability, having its own file might make sense.

It's also very similar to the role component in the admin app. If we end up with a third instance of it somewhere it might make sense to create a shared version.